ctapipe/tools/dl1_merge.py

```diff
@@ -0,0 +1,359 @@
+"""
+Merge DL1-files from stage1-process tool
+"""
+from pathlib import Path
+import sys
+import tables
+import os
+from argparse import ArgumentParser
+import numpy as np
+
+from tqdm import tqdm
+
+from ..core import Provenance
+from ctapipe.core import Tool, traits
+from traitlets import List
+from ctapipe.instrument import SubarrayDescription
+
+import warnings
+
+warnings.filterwarnings("ignore", category=tables.NaturalNameWarning)
+
+PROV = Provenance()
+
+required_nodes = [
+    "/dl1/monitoring/subarray/pointing",
+    "/dl1/monitoring/telescope/pointing",
+    "/dl1/service/image_statistics",
+    "/dl1/service/image_statistics.__table_column_meta__",
+    "/dl1/event/subarray/trigger",
+    "/dl1/event/telescope/trigger",
+    "/dl1/event/telescope/parameters",
```

Comment:
Both images and parameters should also be optional, otherwise we cannot merge files that were created without them. (only parameters / only images)

Reply:
So that means it wouldn't be bad if we want to merge images/parameters, but they are missing in some files? It would make a log.warning but continues with merging?

I was thinking at first of something like: If you know, you want to merge files that were created without images/parameters, then you have to flag --skip_images/ --skip_parameters otherwise it will fail (now).
Or could it happen that somehow you want to merge files were (e.g.) half of them are without images/parameters?

Reply:
It could always happen that a run did not trigger any telescopes (especially with small proton files). So this should probably be not an error case.

Reply:
Ah ok that makes sense.